git/00.setup.md

```diff
@@ -2,9 +2,14 @@
 
 ## 0. Setting up git
 
-If git is not installed it can be install via homebrew or by downloading the XCode command line tools.  The homebrew installation is listed below.
+If git is not installed it can be install via homebrew or by downloading the
```

Comment:
OK. I prefer wrapped lines—multiple changed lines is easier for me to deal with than really long lines—but it's not a big deal to me. Want me to change the PRs to un-wrap the lines?